Makefile

```diff
@@ -150,8 +150,6 @@ deploy: manifests tools ## Deploy controller in the configured Kubernetes cluste
 
 manifests: ## Generate manifests e.g. CRD, RBAC etc.
 	$(MAKE) generate-manifests CONTROLLER_GEN_SRC=./apis/...
-	$(MAKE) -C apis/cpi generate-manifests CONTROLLER_GEN_SRC=./...
-	$(MAKE) -C apis/cni generate-manifests CONTROLLER_GEN_SRC=./...
```

Comment:
question: wondering if still need those code-gen manifests-gen stuff for cpi and cni ? Like `-C apis/addonconfigs/cni` ?

Reply:
nvm, I see a new makefile gets created for cni and cpi under `apis/addonconfigs`